docs/docs/faqs.md

```diff
@@ -115,3 +115,18 @@ Modules can be frozen by setting their `._compiled` attribute to be True, indica
 If you're dealing with "context too long" errors in DSPy, you're likely using DSPy optimizers to include demonstrations within your prompt, and this is exceeding your current context window. Try reducing these parameters (e.g. `max_bootstrapped_demos` and `max_labeled_demos`). Additionally, you can also reduce the number of retrieved passages/docs/embeddings to ensure your prompt is fitting within your model context length.
 
 A more general fix is simply increasing the number of `max_tokens` specified to the LM request (e.g. `lm = dspy.OpenAI(model = ..., max_tokens = ...`).
+
+## Set Vorbose
```

Comment:
typo: verbose

probably we can go with "Set Verbosity Level"

Reply:
Good catch, fixed